typescript/packages/subsurface-viewer/src/SubsurfaceViewer.stories.tsx

```diff
@@ -445,3 +481,61 @@ ViewStateSynchronization.argTypes = {
         control: "check",
     },
 };
+
+export const IsLoadedCallback = (args) => {
+    const [layers, setLayers] = React.useState([defaultWellsLayer]);
+    const [label, setLabel] = React.useState("");
+
+    const handleChange = () => {
+        if (layers.length === 1) {
+            setLayers([meshMapLayerBig, defaultWellsLayer]);
+        } else {
+            setLayers([defaultWellsLayer]);
+        }
+    };
+
+    const props = {
+        ...args,
+        isLoadedCallback: (isLoaded) => {
+            console.log("isLoadedCallback", isLoaded);
+            setLabel(isLoaded ? "LOADED" : "NOT LOADED");
+            return;
+        },
+        layers,
+    };
+
+    return (
+        <Root>
+            <div className={classes.main}>
+                <SubsurfaceViewer {...props} />
+            </div>
+            <button onClick={handleChange}> Add layer </button>
```

Comment:
A bit confusing that this button both adds and removes a layer when it always says "Add layer".